graphql-cli/src/test/resources/expectedGenCode/client/config_types.bal

```diff
@@ -0,0 +1,302 @@
+// Copyright (c) 2022 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+# Client configuration details.
+@display {label: "Connection Config"}
+public type ConnectionConfig record {|
+    # The HTTP version understood by the client
+    HttpVersion httpVersion = HTTP_V2_0;
+    # Configurations related to HTTP/1.x protocol
+    ClientHttp1Settings http1Settings = {};
+    # Configurations related to HTTP/2 protocol
+    ClientHttp2Settings http2Settings = {};
+    # The maximum time to wait (in seconds) for a response before closing the connection
+    decimal timeout = 60;
+    # The choice of setting `forwarded`/`x-forwarded` header
+    string forwarded = "disable";
+    # Configurations associated with request pooling
+    PoolConfiguration poolConfig?;
+    # HTTP caching related configurations
+    CacheConfig cache = {};
+    # Specifies the way of handling compression (`accept-encoding`) header
+    Compression compression = COMPRESSION_AUTO;
+    # Configurations associated with the behaviour of the Circuit Breaker
+    CircuitBreakerConfig circuitBreaker?;
+    # Configurations associated with retrying
+    RetryConfig retryConfig?;
+    # Configurations associated with inbound response size limits
+    ResponseLimitConfigs responseLimits = {};
+    #SSL/TLS-related options
```

Comment:
Let's add a space after `#`

Reply:
Fixed in https://github.com/ballerina-platform/graphql-tools/commit/eb969edf9f95622fe8d6dc7ecc000346998b403e